api/specs.proto

```diff
@@ -122,6 +123,17 @@ message NetworkAttachmentSpec {
 	string container_id = 1 [(gogoproto.customname) = "ContainerID"];
 }
 
+// PluginSpec specifies runtime parameters for a plugin.
+message PluginSpec {
+	// image defines the image reference, as specified in the
+	// distribution/reference package. This may include a registry host, name,
+	// tag or digest.
+	string image = 1;
+
+	// enabled defines if the plugin should be enabled or not.
+	bool enabled = 2;
```

Comment:
We want to enable plugins by default because:
1. After a `docker plugin install`, subsystems/containers can start using the plugin right away. Zero configuration model.
2. Every restart of docker-engine will automatically start plugins alongside, so that containers dont have ordering dependency on plugins lifecycle.

In the event that plugins are deemed to be buggy or unnecessary, they are disabled.